bitwarden/.README.j2

```diff
@@ -1,4 +1,4 @@
-# Home Assistant Community Add-on: Bitwarden RS
+# Home Assistant Community Add-on: Bitwarden (Vaultwarden)
```

Comment:
Shouldn't we just call it Vaultwarden, if their goal is to avoid confusion with Bitwarden itself?,  appreciate that users may get confused if it pops up as an update with a different name.